central/pruning/pruning.go

```diff
@@ -311,6 +314,7 @@ func (g *garbageCollectorImpl) removeOrphanedK8SRoleBindings(searchQuery *v1.Que
 }
 
 func (g *garbageCollectorImpl) removeOrphanedResources() {
+	defer metrics.SetPruningDuration(time.Now(), ops.Remove, "Orphaned Resources")
```

Comment:
I would remove this one as you calculate each one individually which is probably more important.